docs/release-notes.rst

```diff
@@ -5,6 +5,205 @@ This document describes major installer-related changes in Fedora releases.
 
 A guide on adding new entries is in the release documentation.
 
+Fedora 40
+#########
+
+Changes in the graphical interface
+----------------------------------
+
+Dir and image installations run only in the non-interactive text mode now
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Anaconda now requires a fully defined kickstart file for installations into a local image
+(via the ``--image`` cmdline option) or a local directory (via the ``--dirinstall`` cmdline
+option) and these installations can run only in a non-interactive text-based user interface.
+The ``anaconda`` and ``livemedia-creator`` tools can be used for these types of installations
+with the following changes:
+
+    * If a user requests a dir or image installation, Anaconda runs in the text mode.
+    * If the user doesn't specify a kickstart file, Anaconda reports an error and aborts.
+    * If the specified kickstart file is incomplete, Anaconda reports an error and aborts.
+    * All options for specifying the user interface are ignored.
```

Comment:
Formatting of the list looks inconsistent with F39 rel notes, I think there should be just
```
- item
```

Reply:
Fixed.